test/lit/fuzz-types/nominal.test

```diff
@@ -20,3 +20,27 @@
 ;; CHECK-NEXT: (type $17 (array (mut (ref $2))))
 ;; CHECK-NEXT: (type $18 (array (ref $10)))
 ;; CHECK-NEXT: (type $19 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64 (mut f32)) $0))
+;; CHECK-NEXT:
+;; CHECK-NEXT: Inhabitable types:
+;; CHECK-NEXT:
+;; CHECK-NEXT: Built 20 types:
+;; CHECK-NEXT: (type $0 (type $0 (struct (field (mut i16) i31ref f32 f32 f64))))
+;; CHECK-NEXT: (type $1 (type $1 (func (param f64 v128))))
+;; CHECK-NEXT: (type $2 (type $2 (struct (field (mut (ref null $19)) f64 arrayref))))
+;; CHECK-NEXT: (type $3 (type $3 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64) $0)))
+;; CHECK-NEXT: (type $4 (type $4 (struct)))
+;; CHECK-NEXT: (type $5 none)
+;; CHECK-NEXT: (type $6 (type $6 (array (mut eqref))))
```

Comment:
The printing here looks wrong: `(type $6 (type $6` etc.

Reply:
Ah yes. That's actually not related to this PR, so I'll fix that separately.